testdata/p4_14_samples_outputs/copy_to_cpu-first.p4

```diff
@@ -26,9 +26,9 @@ struct metadata {
 }
 
 struct headers {
-    @name(".cpu_header")
+    @name(".cpu_header") 
```

Comment:
where do these extra spaces come from?
something is not right, they should not be here.

Reply:
Last change to these reference files seems to be coming from 
https://github.com/p4lang/p4c/pull/2902

Reply:
But that change seems to remove the spaces, whereas you seem to add them

Reply:
I have reverted back the code pushed which included space and pushed the code again. Please check and let me know if any conflicts are still there